environments/staging.rb

```diff
@@ -0,0 +1,17 @@
+# Use this file for staging-specific deployment configs.
+# https://middlemanapp.com/basics/upgrade-v4/#environments-and-changes-to-configure-blocks
+
```

Comment:
Let's go ahead an add `activate :asset_hash` in here. Like in production. I recently did this on my blog too, and it helps when you're making asset-related changes that you want to see updated on the staging server. Less hair-pulling that way. 😉